cico_build_deploy.sh

```diff
@@ -0,0 +1,12 @@
+#!/bin/bash
+
+set -ex
+
+. cico_setup.sh
+
+build_image
+
+./runtests.sh
+
+push_image
+
```

Comment:
There will be no docker image for this library so we can remove `push_image` and `build_image`.